### PR TITLE
Various Updates to ValidateContent Script

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Updates the timeout of the automation to 10 minutes. This should address timeouts when validating enormous packs.
- Updates the docker image tag to the latest version.
- Handles input of single content items that are not integration/automations (aka items that need to be split by the demisto-sdk) including items stored as JSON data e.g. classifiers.

## Minimum version of Cortex XSOAR
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/ValidateContent/ValidateContent.yml [hunk 1]
```diff
@@ -53,10 +53,10 @@ outputs:
   description: The code line number in which the error was found in the lint.
   type: String
 scripttarget: 0
-timeout: 300ns
+timeout: 600ns
 runonce: false
 runas: DBotWeakRole
 fromversion: 5.5.0
-dockerimage: demisto/xsoar-tools:1.0.0.17588
+dockerimage: demisto/xsoar-tools:1.0.0.18207
```
Comment (Itay4) [inline Packs/Base/Scripts/ValidateContent/ValidateContent.yml:60]: lets update with the new docker image that includes sdk 1.3.5

File: Packs/Base/ReleaseNotes/1_10_4.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### ValidateContent
+- Increase the script timeout to 10 minutes.
+- Add support to run validate on single content items of all types.
```
Comment (Itay4) [inline Packs/Base/ReleaseNotes/1_10_4.md:5]: lets mention here the docker image bump

**Review by Itay4 — APPROVED** [review]

File: Packs/Base/ReleaseNotes/1_10_4.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### ValidateContent
+- Increase the script timeout to 10 minutes.
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_10_4.md:4]:
```suggestion
- Increased the timeout for the script to 10 minutes.
```

File: Packs/Base/ReleaseNotes/1_10_4.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+- Add support to run validate on single content items of all types.
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_10_4.md:5]:
```suggestion
- Added support to run validation on single content items of all types.
```

File: Packs/Base/ReleaseNotes/1_10_4.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+- Update the script's docker image tag.
```
Comment (amshamah419) [inline Packs/Base/ReleaseNotes/1_10_4.md:6]:
```suggestion
- Upgraded the Docker image to *demisto/xsoar-tools:1.0.0.19258*.
```

**Pull request merged**